comments/api/tests.py

```diff
@@ -110,3 +114,101 @@ def test_create_api(self):
             response.data['errors']['tweet_id'][0],
             'tweet does not exist.'
         )
+
+    def test_update_api(self):
+        comment = self.create_comment(self.lbj23, self.tweet, 'GOAT!')
+        url = COMMENT_DETAIL_URL.format(comment.id)
+
+        # Non-login users
+        response = self.anonymous_client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
This permission_class might be not suitable when retrieve api is considered.
Can you add some comment on this?

Reply:
Resolved as per suggestion!